OPTS.m

```diff
@@ -21,6 +21,8 @@
         time_independent_build_A = false;
         many_solution_capable = false;
         max_lev = 8;
+        max_lev_coeffs = false; % when enabled, build partial term coeff matrices for some max level,
```

Comment:
let's rename this to `use_max_lev_coeffs` and set the default to `true`. 

I just changed it to default true and 5 of the tests fail. Can you look into that?

Reply:
I guess the part I'm confused about is why the tests pass when we don't use the max_lev_coeffs approach - I assume this is only a problem for implicit time advance (since all these are), so it's really just the same symptom of the ill-conditioned matrix problem, and since large condition number mean small changes get amplified, then we can likely just adjust the tests to use a different range, or just adjust the tolerance - I think adjusting the tolerance on these is OK, since i ran the case with and without the new method and it still "looks good".